tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -80,28 +80,27 @@ using Random
 Random.seed!(1789);
 ```
 
-All packages used across examples are placed here. You can install packages via `Using Pkg; Pkg.add(package_name)`.
+All packages used across examples are placed here. You can install packages via `Using Pkg; Pkg.add("package_name")`.
 
 Package usages:
 
 > We use `DataFrames` for instantiating matrices, `LinearAlgebra` to perform matrix operations;
-> `Turing` for model specification and sampling;
-> `VegaLite` and `StatsPlots` for ploting the resutls.
-> Further, in a later example, we retieeve the *Iris* dataset from `RDatasets`.
-> As all examples involve the sampling, for reproducity purpose, we set fixed seed using the `Random` library.
+> `Turing` for model specification and MCMC sampling;
+> `VegaLite` and `StatsPlots` for visualising the resutls.
+> As all examples involve the sampling, for reproducity purpose, we set fixed seed using the `Random` package.
 
-#### Step 2: Synthetic data generation
+#### Step 2: Data generation
 
 Here, we simulate the biological gene expression problem described earlier.
 We simulate 60 cells, each cell has 9 gene features.
 This is a simplified problem with only a few cells and genes for demonstration purpose, which is not comparable to the complexity in real-life (e.g. thousands of features for each individual).
 Even so, spotting the structures or patterns in a 9-feature space would be a challenging task; it would be nice to reduce the dimentionality using p-PCA.
 
 ```julia
-n_cells = 60
-n_genes = 9
-mu_1 = 10.0 * ones(n_genes ÷ 3)
+n_genes = 9 # D
+n_cells = 60 # N
 mu_0 = zeros(n_genes ÷ 3)
+mu_1 = 10.0 * ones(n_genes ÷ 3)
```

Comment:
This should really use `fill` (but see below for some other suggestion).

Reply:
yes, `fill` is more efficient. However, following your suggestion on simplifying the data generation process, I have replaced the whole code block using the method you suggested, which is much more efficient.